src/messages/manifestjson.js

```diff
@@ -165,3 +165,11 @@ export const NO_DEFAULT_LOCALE = {
     See: https://mzl.la/2hjcaEE`),
   file: MANIFEST_JSON,
 };
+
+export const WRONG_ICON_EXTENSION = {
+  code: 'WRONG_ICON_EXTENSION',
+  legacyCode: null,
+  message: _('Consider Adding another icon'),
```

Comment:
Sorry, I didn't notice this earlier. This message is prominently shown to developers so it should be more explicit.

Something along the lines of `Unsupported image extension` and then the `description` goes into the details.

Mind changing that?